powerhome/playbook
# Remove typos from Tooltip Kit

#### Screens

<img width="1790" alt="Screen Shot 2022-09-08 at 3 54 54 PM" src="https://user-images.githubusercontent.com/8194056/189203133-4e5d1382-a65a-49c6-8647-0bf64fd612ce.png">

#### Breaking Changes

No breaking changes, just remove the `/* @flow*/` and a `console.log` from the kit.

#### Runway Ticket URL

No runway

#### How to test this

Check milano for the rendered component.  Analyze code changes and implement the kit with different props. Check `console.log` for any errors.

#### Checklist:
